.github/workflows/pull-request.yml

```
@@ -328,20 +328,49 @@ jobs:
         run: |
           python .github-central/.github/workflows/scripts/disable_statements_check.py --files ${{ steps.changed-files.outputs.all_changed_files }}
 
-  Check-Test-Quality:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Check for test anti-patterns
-        run: |
-          echo "🔍 Scanning for test quality issues..."
+Check-Test-Quality:
```

actionlint (1.7.11)

[error] unexpected key "Check-Test-Quality" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)